examples/batch/README.md

```diff
@@ -0,0 +1,16 @@
+# Azure Batch Sample
+
+Sample to deploy a Job in Azure Batch
```

Comment:
since this doesn't yet include the ability to provision a Job, can we update the description to reflect that for the moment:

```suggestion
Sample to deploy a Azure Batch instance
```

Reply:
Done.